images/win/scripts/Installers/Install-PowerShellModules.ps1

```diff
@@ -0,0 +1,39 @@
+################################################################################
+##  File:  Install-PowerShellModules.ps1
+##  Desc:  Install PowerShell Modules.
+################################################################################
+
+$ErrorActionPreference = "Stop"
+
+# Set TLS1.2
+[Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor "Tls12"
+
+Write-Host "Setup PowerShellGet"
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+
+# Specifies the installation policy
+Set-PSRepository -InstallationPolicy Trusted -Name PSGallery
+
+# Install PowerShell modules
+$modules = (Get-ToolsetContent).powershellModules
+
+foreach($module in $modules)
+{
+    $moduleName = $module.name
+    Write-Host "Installing ${moduleName} module"
+
+    if ($module.versions)
+    {
+        foreach ($version in $module.versions)
+        {
+            Write-Host " - $version"
+            Install-Module -Name $moduleName -RequiredVersion $version -Scope AllUsers -SkipPublisherCheck -Force
+        }
+        continue
+    }
+
+    Install-Module -Name $moduleName -Scope AllUsers -SkipPublisherCheck -Force
+}
+
+Import-Module Pester

```

Comment:
We need to import Pester explicitly because we install it in the same script, right?

Reply:
We import module that contains `Get-ToolsetContent`  before we had installed Pester module and type [PesterConfiguration] is not found in current sessions.

```
PS > [PesterConfiguration]; Import-Module Pester;[PesterConfiguration]
Unable to find type [PesterConfiguration].
At line:1 char:1
+ [PesterConfiguration]; Import-Module Pester;[PesterConfiguration]
+ ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (PesterConfiguration:TypeName) [], RuntimeException
    + FullyQualifiedErrorId : TypeNotFound


IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     False    PesterConfiguration                      System.Object
```